mu-plugins/plugin-loader.php

```diff
@@ -15,6 +15,7 @@
  */
 function create_wordpress_project_core_plugins(): array {
 	return [
+		// INSTALLED_PLUGINS_PLACEHOLDER.
```

Comment:
I also wonder if it'd be a better experience here to include the default set of plugins and then replace it with the configuration script. 

When working on the configuration scripts for the `create-*` projects I always strived to have a working starter state. In a perfect world we'd have the default plugins included here and then the configuration script could fix/adjust it rather than leaving it missing here.